assets/prometheus-k8s/prometheus-k8s-service-monitor-logging-elasticsearch.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: logging-elasticsearch
+  labels:
+    k8s-app: logging-elasticsearch
+spec:
+  selector:
+    matchLabels:
+      k8s-app: logging-elasticsearch
+      prometheus: TBD
+  namespaceSelector:
+    matchNames:
+    - this-namespace-is-automatically-replaced-at-runtime
```

Comment:
This should be an openshift-* namespace

Reply:
However - if we want to use this with OCP 3.9 and earlier, we will have to be able to use namespace `logging` for Elasticsearch and the other logging components, and `openshift-logging` in OCP 3.10 and later.

Reply:
Cluster Monitoring Operator is designed for OCP 3.10+

Reply:
@ironcladlou is correct, it must be fixed to openshift-logging and we should not allow any choice.